backend/metrics_backend/utils/serialisation.py

```diff
@@ -38,13 +48,33 @@ def token_network_to_dict(token_network: TokenNetwork) -> Dict:
 
         if view.state == ChannelView.State.OPENED:
             num_channels_opened += 1
```

Comment:
This information is in the MatricState, right?

Reply:
see comment above